setup.py

```diff
@@ -0,0 +1,140 @@
+RE_TITLE = re.compile(r'^(?P<level>#+)\s*(?P<title>.*)$', re.M)
+
+BADGES_TO_KEEP = ['gitter-badge']
+
+RST_TITLE_LEVELS = ['=', '-', '*']
+
+RST_BADGE = '''\
+.. image:: {badge}
+    :target: {target}
+    :alt: {text}
+'''
+
+
+def md2pypi(filename):
+    '''
+    Load .md (markdown) file and sanitize it for PyPI.
+    Remove unsupported github tags:
+     - code-block directive
+     - travis ci build badges
+    '''
+    content = io.open(filename).read()
+
+    for match in RE_MD_CODE_BLOCK.finditer(content):
+        rst_block = '\n'.join(
+            ['.. code-block:: {language}'.format(**match.groupdict()), ''] +
+            ['    {0}'.format(l) for l in match.group('lines').split('\n')] +
+            ['']
+        )
+        content = content.replace(match.group(0), rst_block)
+
+    refs = dict(RE_LINK_REF.findall(content))
+    content = RE_LINK_REF.sub('.. _\g<key>: \g<url>', content)
+    content = RE_SELF_LINK.sub('`\g<1>`_', content)
+    content = RE_LINK_TO_URL.sub('`\g<text> <\g<url>>`_', content)
+
+    for match in RE_BADGE.finditer(content):
+        if match.group('badge') not in BADGES_TO_KEEP:
+            content = content.replace(match.group(0), '')
+        else:
+            params = match.groupdict()
+            params['badge'] = refs[match.group('badge')]
+            params['target'] = refs[match.group('target')]
+            content = content.replace(match.group(0),
+                                      RST_BADGE.format(**params))
+    # Must occur after badges
+    for match in RE_LINK_TO_REF.finditer(content):
+        content = content.replace(match.group(0), '`{text} <{url}>`_'.format(
+            text=match.group('text'),
+            url=refs[match.group('ref')]
+        ))
+
+    for match in RE_TITLE.finditer(content):
+        underchar = RST_TITLE_LEVELS[len(match.group('level')) - 1]
+        title = match.group('title')
+        underline = underchar * len(title)
+
+        full_title = '\n'.join((title, underline))
+        content = content.replace(match.group(0), full_title)
+
+    return content
+
+
+def pip(filename):
+    '''Parse pip reqs file and transform it to setuptools requirements.'''
+    requirements = []
+    for line in open(os.path.join('requirements', filename)):
+        line = line.strip()
+        if not line or '://' in line or line.startswith('#'):
+            continue
+        requirements.append(line)
+    return requirements
+
+
+long_description = '\n'.join((
+    md2pypi('README.md'),
+    md2pypi('CHANGELOG.md'),
+    ''
+))
+
+install_requires = pip('install.pip')
+tests_require = pip('test.pip')
+
+
+setup(
+    name='udata-recommendations',
+    version=__import__('udata_recommendations').__version__,
+    description=__import__('udata_recommendations').__description__,
+    long_description=long_description,
+    url='https://github.com/opendatateam/udata-recommendations',
+    author='Opendata Team',
+    author_email='opendatateam@data.gouv.fr',
```

Comment:
Use `contact@opendata.team` which is more neutral (and currently an alias to `opendatateam@data.gouv.fr`)

Reply:
👍